helmexec/helmexec.go

```diff
@@ -9,4 +9,6 @@ type Interface interface {
 	SyncRelease(name, chart string, flags ...string) error
 	DiffRelease(name, chart string, flags ...string) error
 	DeleteRelease(name string) error
+
+	SecretDecrypt(name string) (string, error)
```

Comment:
Very nit but I slightly prefer `DecryptSecret` so that it looks consistent with other functions!